### ISSUE TITLE
zoho in rcx not available

### ISSUE THREAD

**Issue by pflegende** [rclone/rclone#5937]
#### What is the problem you are having with rclone?

```
@njcw
said zoho should be available in rcx.

but doesnt

https://twitter.com/njcw/status/1484621836021563393?t=mTm_6Zl1JJ1ULrl1R_dbmw&s=19
```

#### What is your rclone version (output from `rclone version`)

```
rclone 1.51.0
rcx app version 1.11.4
```

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

```
android 10.0.0.195
```

#### Which cloud storage system are you using? (e.g. Google Drive)

```
try zoho, but is not listed 
```

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

```
add 

aka "+" on rcx dialo screen
```

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

```
because is no activity available for zoho, a log seems not usefull
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by animosity22** [comment]
Update rclone.

Please use forums for questions....

**Comment by pflegende** [comment]
We do so.

BUT:
We wrote it as an issue because of this post of the Mainainer @ncw on Twitter:
https://twitter.com/njcw/status/1484621836021563393?t=mTm_6Zl1JJ1ULrl1R_dbmw&s=19

The post of the maintainer, who said to put in rclones **ISSUE** really.

So we close it.

happy weekend
:)

**Comment by pflegende** [comment]
We understand @ncw in his Twitter post in such a cause:
zoho should be available in rcx
and if it is not available it is a bug
aka a issue on github
what he has write down also in his twitter post.

Bur, not at all.
let the issue be closed.
We go to the forum.
May be we understand the hint of @ncw in false manner.

**Comment by animosity22** [comment]
It isn't an issue because you aren't using the latest version as you have an old version which is why it isn't listed.

He wrote to report any bugs here, which this isn't.

If you read the reporting the issue, you can see all that information.

**_Please show the effort you've put into solving the problem and please be specific.
People are volunteering their time to help! Low effort posts are not likely to get good answers!

If you think you might have found a bug, try to replicate it with the latest beta (or stable).
The update instructions are available at https://rclone.org/commands/rclone_selfupdate/

If you can still replicate it or just got a question then please use the rclone forum:

    https://forum.rclone.org/

for a quick response instead of filing an issue on this repo._**

**Comment by pflegende** [comment]
Great! thx!
update in progression

sorry for some effort not needed.

thx again

**Comment by pflegende** [comment]
@animosity22 commented 4 days ago
`It isn't an issue because you aren't using the latest version as you have an old version which is why it isn't listed.
He wrote to report any bugs here, which this isn't.`

We used the with `latest` labeled version of rcx and the behavior is consistent: Zoho is not listed in the availabale "add" (aka "+") Services. 

result:
- latest version of rcx
- it seems the requirements given by @animosity22 are solved what he defiend a issue: we used the latest version (by the way: the rclone version itself could not be choosen by the rcx-user. The user can only choose the rcx-version. here: the "latest".)
- so we opened it again as an ISSUE: `zoho in rcx not available`
- otherwise all is decribed in our initial issue-description

:)
